README.md

```diff
@@ -339,20 +339,26 @@ Returns a response object.
 The official [Payflow Gateway Developer Guide and
 Reference](https://developer.paypal.com/docs/classic/payflow/integration-guide/)
 
+# SUPPORT
+
+Bugs may be submitted through [https://github.com/maxmind/webservice-paypal-paymentsadvanced/issues](https://github.com/maxmind/webservice-paypal-paymentsadvanced/issues).
+
 # AUTHOR
 
 Olaf Alders <olaf@wundercounter.com>
 
 # CONTRIBUTORS
 
+- Andy Jack <andyjack@users.noreply.github.com>
```

Comment:
Should this use Andy's MaxMind email instead? Looks like this might be automatically generated though huh?